lint/file.go

```diff
@@ -99,9 +97,12 @@ func (f *File) isMain() bool {
 	return false
 }
 
+const directiveSpecifyDisableReason = "specify-disable-reason"
+
 func (f *File) lint(rules []Rule, config Config, failures chan Failure) {
 	rulesConfig := config.Rules
-	disabledIntervals := f.disabledIntervals(rules, config.SpecifyDisableReason)
+	_, mustSpecifyDisableReason := config.Directives[directiveSpecifyDisableReason]
```

Comment:
Do we handle the case when `directiveSpecifyDisableReason` is not specified in the config?

Reply:
Yes, after the execution of this line, if `directiveSpecifyDisableReason` is not in the config, the variable `mustSpecifyDisableReason` is `false`